### PR TITLE
bay clicky noises

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Adds the typing noises when clicking a computer, switch, or sleeper

<s>Also adds the computer sprites from @Quantum-M</s> gonb
## Why It's Good For The Game

I think its very atmospheric, and helps the atmosphere of the game

## Changelog
:cl:
soundadd: bay clicking sounds
/:cl:

### CODE REVIEW

File: code/game/sound.dm [hunk 1]
```diff
@@ -272,4 +272,12 @@ distance_multiplier - Can be used to multiply the distance at which the sound is
...
+			if("keystroke")
+				soundin = pick('sound/machines/keyboard/keypress1.ogg','sound/machines/keyboard/keypress2.ogg','sound/machines/keyboard/keypress3.ogg','sound/machines/keyboard/keypress4.ogg')
+			if("keyboard")
+				soundin = pick('sound/machines/keyboard/keystroke1.ogg','sound/machines/keyboard/keystroke2.ogg','sound/machines/keyboard/keystroke3.ogg','sound/machines/keyboard/keystroke4.ogg')
+			if("button")
+				soundin = pick('sound/machines/button1.ogg','sound/machines/button2.ogg','sound/machines/button3.ogg','sound/machines/button4.ogg')
+			if("switch")	//stolen from nsv
```
Comment (ZephyrTFA) [inline code/game/sound.dm:281]: Did you obtain permission to use their sound file, or is it under an applicable free use copyright?

Comment (rye-rice) [reply code/game/sound.dm:281]: the keyboard sounds were taken from https://github.com/Baystation12/Baystation12/commit/1caacb42ffe261df87f151384e8e4c78b93e23d5#diff-dd29442deca69f52c50006b831cb216edf78a7da33748f0a80ff19f2ebe57ecd
and the 'switch' sounds were taken from https://github.com/BeeStation/NSV13/commit/e3540f769f926678764426ec267c46d38a3e10da#diff-0f957de0c8438357260c3745900bc6b0e47261415aaad228c7de51c1c4ea1616 which actually doesn't list where they got them from... which is slightly concerning now im thinking about it

Comment (ZephyrTFA) [reply code/game/sound.dm:281]: Commit message indicates they're free source sound files. All good then.

**Comment by ghost** [comment]
My criticism is that the sprites really need to be made smaller, and also you need to have sprites for computer construction.

**Comment by rye-rice** [comment]
Quantum is working on improving them at the moment

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: aabc6f1b144d0c36cf9f9f5d49313c6586ec6706
Pull Request: 281b0945b6ffcbd0351001f87a75b705fb10cca8
Server: 4a612b6abc77d3aaae270b65dc4c94667e4d6be2

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 293bad3e54aa3f26644d5d871baee9eda36057ca
Pull Request: 281b0945b6ffcbd0351001f87a75b705fb10cca8
Server: 2c17e761a3dbe846412b0eb5224178d8c7dfc1e4

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: d79659920aec58007df63d3e9b96cb0fe1a16353
Pull Request: 281b0945b6ffcbd0351001f87a75b705fb10cca8
Server: 72d94bef2620899bd315f660ccb547df14ffc328

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: dd854e7386a7a9a664d6357aab6213b032e1817b
Pull Request: 281b0945b6ffcbd0351001f87a75b705fb10cca8
Server: d097fa6e32a13458e756307a43c1764f99b243bc

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 4d98cdcea7b4591b94a06237309039f4980217fa
Pull Request: 281b0945b6ffcbd0351001f87a75b705fb10cca8
Server: 9a29cfa82da92d64f18a8f8bcd69d3ea934be1ac

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: f789e74004b3acb82690d77398848b240a153b6f
Pull Request: 281b0945b6ffcbd0351001f87a75b705fb10cca8
Server: fec71ade6e716fb63d267abca95b86e22f5eb182

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: bf2a3d7daffd8042d3adbe075ec803cd3abd4e03
Pull Request: 281b0945b6ffcbd0351001f87a75b705fb10cca8
Server: ca0ff8b9f76846976a9d8d2ec5e18cf60ecd28be

**Comment by TheNeoGamer42** [comment]
empty computer frames are completely invisible right now

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by rye-rice** [comment]
i forgot i made this pr

**Review by MarkSuckerberg — APPROVED** [review]

**Comment by rye-rice** [comment]
i swear i was going to add something else to this pr but i forgot

**Comment by rye-rice** [comment]
you can merge it if you want

**Pull request merged**